README.md

```diff
@@ -425,17 +424,25 @@ This message is used to communicate the current sonar state. If the data field i
 ### control
 
 #### 2600 reset
-Reset the sonar. The bootloader may run depending on the selection according to the `bootloader` payload field. When the bootloader runs, the external LED flashes at 5Hz. If the bootloader is not contacted within 3 seconds, it will run the current program. If there is no program, then the bootloader will wait forever for a connection.
+Reset the sonar. The bootloader may run depending on the selection according to the `bootloader` payload field. When the bootloader runs, the external LED flashes at 5Hz. If the bootloader is not contacted within 5 seconds, it will run the current program. If there is no program, then the bootloader will wait forever for a connection. Note that if you issue a reset then you will have to close all your open comm ports and go back to issuing either a discovery message for UDP or go through the break sequence for serial comms before you can talk to the sonar again.
 
 | Type | Name             | Description                                                      | Units |
 |------|------------------|------------------------------------------------------------------|-------|
 | u8 | bootloader | 0 = skip bootloader; 1 = run bootloader |  |
+| u8 | reserved | 0 = skip bootloader; 1 = run bootloader |  |
```

Comment:
The description does not match. `Reserved for additional features in the future.`